### PR TITLE
Two test cases created

### PR DESCRIPTION
**Pull request by qlinsey**
1. ResultTupleTest 
2. ArrowRowViewTest

### CODE REVIEW

**Comment by qlinsey** [comment]
Hi, thanks. 
(1) For: Add '\n' in the last line of files. Question: Do you have some format config for java within Intellji ? for instance: ( https://scalameta.org/scalafmt/docs/installation.html#intellij ) . Can pls point me to the java format config for Intellij if this project is using so that I can set up on my local? thanks . 
(2) I will check. thanks

**Comment by Zrealshadow** [comment]
> Hi, thanks. (1) For: Add '\n' in the last line of files. Question: Do you have some format config for java within Intellji ? for instance: ( https://scalameta.org/scalafmt/docs/installation.html#intellij ) . Can pls point me to the java format config for Intellij if this project is using so that I can set up on my local? thanks . (2) I will check. thanks

We use the eclipse default format config. It's a default option in intellij.
As for new line in the end of file, it's not format issue. The format tool will not help you add new line at the end of the file automatically.
this [url](https://stackoverflow.com/questions/5813311/whats-the-significance-of-the-no-newline-at-end-of-file-log) explains why we should do it.

**Comment by qlinsey** [comment]
Thanks, will read and update these days.

File: cool-extensions/arrow-extensions/src/test/com/nus/cool/extension/util/arrow/ArrowRowViewTest.java [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+    public void testValid(VectorSchemaRoot root) {
+        ArrowRowView arv1 = new ArrowRowView(root,1);
+        Assert.assertTrue(arv1.valid());
+        ArrowRowView arv2 = new ArrowRowView(root,4);
+        Assert.assertFalse(arv2.valid());
+    }
+
+    @Test(dataProvider = "ArrowRowReviewDP")
+    public void testGetField(VectorSchemaRoot root) {
+        {
```
Comment (Zrealshadow) [inline cool-extensions/arrow-extensions/src/test/com/nus/cool/extension/util/arrow/ArrowRowViewTest.java:33]: extra curly brackets ?

**Review by Zrealshadow — APPROVED** [review]
Delete the extra curly brackets, we can merge this pr

**Comment by KimballCai** [comment]
We will first merge all updates into the `dev` branch for developments and tests. Once the system is updated into a new version, we will merge it into the main branch. Thanks

**Comment by qlinsey** [comment]
No problem.

**Comment by KimballCai** [comment]
have tested and can merge into the dev branch.

**Review by KimballCai — APPROVED** [review]
can merge

**Pull request merged**